tools/bioformats2raw/.shed.yml

```diff
@@ -4,6 +4,6 @@ categories:
 description: Convert image to OME-Zarr
 long_description: Convert image to OME-Zarr using bioformats2raw
 name: bioformats2raw
-owner: euro-bioimaging
+owner: imgteam
 homepage_url: https://github.com/Euro-BioImaging
-remote_repository_url: https://github.com/Euro-BioImaging/Galaxy-OME-Zarr/tree/main/tools/bioformats2raw
\ No newline at end of file
+remote_repository_url: https://github.com/Euro-BioImaging/Galaxy-OME-Zarr/tree/main/tools/bioformats2raw
```

Comment:
This URL is also not correct, but can be changed later on.

Reply:
Will do that.